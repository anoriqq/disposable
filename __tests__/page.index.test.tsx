import React from 'react';
import renderer from 'react-test-renderer';
import Index from '../pages/index';

jest.useFakeTimers();

it('renders homepage unchanged', () => {
  const tree = renderer.create(<Index />).toJSON();
  expect(tree).toMatchSnapshot();
});
