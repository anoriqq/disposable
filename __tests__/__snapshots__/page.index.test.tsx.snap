// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<a
  href="/auth"
>
  login
</a>
`;
